redom/api/src/main/kotlin/com/pushtorefresh/redom/api/ViewStructure.kt

```diff
@@ -1,27 +1,18 @@
 package com.pushtorefresh.redom.api
 
 sealed class ViewStructure {
+
+    abstract val clazz: Class<out com.pushtorefresh.redom.api.View>
+    abstract val layoutParams: LayoutParams?
```

Comment:
We've discussed this, seems to be a bit dangerous due to potential changes to layoutParams made from user code directly on view which can result in Redom/RecyclerView reusing wrong view #62